stats/parsing-stats/lang/scala/projects.txt

```diff
@@ -8,7 +8,7 @@ https://github.com/akka/akka
 https://github.com/yahoo/CMAK
 https://github.com/ornicar/lila
 https://github.com/gitbucket/gitbucket
-https://github.com/twitter/finagle
+# https://github.com/twitter/finagle
```

Comment:
Could you justify this or remove the change?

Reply:
that test causes a segfault on my machine, i usually comment it for testing purposes and occasionally i forget and accidentally commit it (that's what happened here)

Reply:
That's unsettling